ONSdigital/response-operations-ui
# Make collection instrument call optional

# What and why?

On the `/surveys/<surveyshortname>/period` page, it makes 2 necessary calls to collection instrument which are probably a hangover from back in the day when we used to do collection instrument operations on this page.

We've made the instrument details for the exercise optional and opted out of getting them for just that one page only.

There is probably more work to do in this area but this is a temporary fix to smooth out an issue with a large number of  BRES instruments timing out the page.

# How to test?

- Deploy to environment
- Seed test data with acceptance-tests
- Verify live exercise loads correctly
- Create new collection exercise
- Begin setting it up and make sure that nothing goes wrong.  Functionally this should be the same, only with less calls to get the same thing.

# Trello
